modules/ROOT/pages/_partials/access-mgmt-steps-partial.adoc

```diff
@@ -0,0 +1,3 @@
+. Log in to Anypoint Platform using an account that has the `Organization Administrator` permission.
+. In the navigation bar or the main Anypoint Platform page, click *Access Management*.
```

Comment:
I think we decided on "sidebar" for the navigation bar.

Reply:
This is in Sara's access management doc. I will let her know.